<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.philipfranchi.hibernateValidator.hibernateValidatorTest" tests="2" skipped="0" failures="0" errors="0" timestamp="2018-01-31T21:39:51" hostname="nympen54232.us.randomhouse.com" time="0.146">
  <properties/>
  <testcase name="shouldFailWithNameNotInSet" classname="com.philipfranchi.hibernateValidator.hibernateValidatorTest" time="0.145"/>
  <testcase name="shouldSucceedWithGoodSet" classname="com.philipfranchi.hibernateValidator.hibernateValidatorTest" time="0.001"/>
  <system-out><![CDATA[String is not in set {a,b,c}
]]></system-out>
  <system-err><![CDATA[Jan 31, 2018 4:39:51 PM org.hibernate.validator.internal.util.Version <clinit>
INFO: HV000001: Hibernate Validator 5.4.1.Final
]]></system-err>
</testsuite>
